canonical-web-and-design/maas-ui
# refactor(ui): refactor machine system card to work with controllers

## Done

- Refactored machine system card to work with controllers
- Slightly updated design and renamed to `HardwareCard`

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to the details page of a machine and check that the hardware card [matches the design](https://docs.google.com/document/d/1H-GWCLx3aSRiJRo9h5Fz-6IFLQ3adsa63pRFGlBFuq0/edit#heading=h.gczn4qs17xmf)

## Fixes

Fixes canonical-web-and-design/app-tribe#1001

## Screenshots

![Screenshot 2022-06-02 at 12-13-13 alive-teal maas details ⚠️ bolla MAAS](https://user-images.githubusercontent.com/25733845/171545587-d1e0207b-ffef-4fe1-acc8-f295852daa1c.png)
